modules/internal/ChapelBase.chpl

```diff
@@ -1485,9 +1511,7 @@ module ChapelBase {
 
   // non-param/param and param/non-param
   inline proc %(a: uint(64), param b: uint(64)) {
-    return __primitive("%", a, b);
-  }
-  inline proc %(param a: uint(64), b: uint(64)) {
+    if b == 0 then compilerError("Attempt to divide by zero");
```

Comment:
It seems to me that we should have a:

````chapel
inline proc %(a: int(64), param b: int(64)) { ... }
````

version as well for symmetry (and since ints can't coerce to uints).